db/queries.ts

```diff
@@ -0,0 +1,67 @@
+import { drizzle } from "drizzle-orm/postgres-js";
+import { desc, eq } from "drizzle-orm";
+import postgres from "postgres";
```

Comment:
i'm getting some eslint errors here we should look into

Reply:
`drizzle-orm` import should occur before import of `drizzle-orm/postgres-js`
There should be at least one empty line between import groups